Gemfile

```diff
@@ -29,8 +29,11 @@ group(:development, :test) do
   gem 'rake', require: false
   gem 'rspec', '~> 3.0', require: false
   gem 'rubocop', '~> 1.0', require: false
+  gem 'rubocop-capybara', require: false
+  gem 'rubocop-factory_bot', require: false
   gem 'rubocop-rake', require: false
   gem 'rubocop-rspec', require: false
+  gem 'rubocop-rspec_rails', require: false
```

Comment:
Why did you add that rubocop plugin? Isn't that specific to rails applications?

Reply:
@bastelfreak 
We need the gem because it seems we are using some rails cops in vanagon which have now been moved to the 'rubocop-rspec_rails' gem: https://github.com/puppetlabs/vanagon/blob/main/.rubocop.yml#L419C1-L432C16.

Reply:
This might be a case where we need to remove the cops from rubocop.yml. I don't believe vanagon depends on any rails functionality. It would be very surprising if it did.